src/js/components/PodInstancesTable.js

```diff
@@ -281,6 +290,31 @@ class PodInstancesTable extends React.Component {
     );
   }
 
+  renderColumnLogs(prop, row, rowOptions = {}) {
+    if (rowOptions.isParent) {
+      return <Icon id="" size="mini" />;
```

Comment:
You could try something like `<div>&nbsp;</div>` since the height is defined by the line height. However I would like a confirmation from the other guys, since we tend to avoid non-breaking space.